### PR TITLE
Bugfix:CVP: Update the operator-scorecard sdk image mapping

### PR DESCRIPTION
**Pull request by samvarankashyap**
$subject
Need to update the mapping of the operator scorecard image from cvp

### CODE REVIEW

**Review by nmars — APPROVED** [review]
LGTM

File: clusters/app.ci/supplemental-ci-images/cvp-operator-scorecard.yaml [hunk 1]
```diff
@@ -8,6 +8,6 @@ spec:
   - name: v1
     from:
       kind: DockerImage
-      name: quay.io/cvpops/operator-scorecard:v5
+      name: quay.io/cvpops/operator-scorecard:v6
```
[the remark this answers is not in the document]
Comment (samvarankashyap) [reply clusters/app.ci/supplemental-ci-images/cvp-operator-scorecard.yaml:11]: @hongkailiu Do you want me to remove the whole mapping file ? 
am not sure about the syntax

[the remark this answers is not in the document]
Comment (samvarankashyap) [reply clusters/app.ci/supplemental-ci-images/cvp-operator-scorecard.yaml:11]: @hongkailiu We would like to map it to  quay.io/cvpops/operator-scorecard:v6 ? 
its not mentioned in supplemental_ci_images_ci

[the remark this answers is not in the document]
Comment (hongkailiu) [reply clusters/app.ci/supplemental-ci-images/cvp-operator-scorecard.yaml:11]: I meant we can remove the `spec` in this ImageStream.
The mapping file is enough to do the mirroring.

**Pull request merged**